youtube_dlc/downloader/external.py

```diff
@@ -5,6 +5,13 @@
 import subprocess
 import sys
 import time
+import shutil
+
+try:
+    from Crypto.Cipher import AES
+    can_decrypt_frag = True
```

Comment:
should be `_can_decrypt_frag` unless you are accessing it from other files

Reply:
Following the same structure as hls.py

Reply:
hm.. I dont agree that it is the correct coding convention, but you are right. No need to change it